tests/pypy2_bugs.txt

```diff
@@ -1,7 +1,6 @@
 # Specific bugs that only apply to pypy2
 
 build.cythonize_script
```

Comment:
I think this'll be blocking ` build.cythonize_script_package` - these are blocked if it matches part of it, rather than matches all of it.

Reply:
ok